_includes/tutorials/error-handling/kstreams/markup/dev/make-exception-handler.adoc

```diff
@@ -65,4 +65,10 @@ Now create the following file at `src/main/java/io/confluent/developer/MaxFailur
 
 You'll add the `StreamsUncaughtExceptionHandler` to your Kafka Streams application in the next step.
 
+There's one more step you'll need to take and that is creating a custom exception class `ProcessorException` that your exception handler will process. Create this simple Java class at `src/main/java/io/confluent/developer/ProcessorException.java`
```

Comment:
@bbejeck what's the tl;dr on why this change helps (whereas before not needed)?

Reply:
in a nutshell, Kafka Streams now takes a look at any uncaught exception. If it's an `IllegalStateException`  or `IllegalArgumentException` (both standard exceptions in the Java API), the user won't handle those and the application shuts down.  The KT previously threw an `IllegalStateException` but that wasn't the best choice as it's a better practice to create an exception that has some business context which is the solution to fixing this KT.